olive/cli/cloud_cache.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from olive.cli.base import BaseOliveCLICommand
+from olive.common.utils import get_credentials
+
+logger = logging.getLogger(__name__)
+
+
+class CloudCacheCommand(BaseOliveCLICommand):
+    @staticmethod
+    def register_subcommand(parser):
+        sub_parser = parser.add_parser("cloud-cache", help="Cloud cache model operations")
+        sub_parser.add_argument(
+            "--delete",
+            action="store_true",
+            help="Delete a model cache from the cloud cache.",
+        )
+        sub_parser.add_argument(
+            "--account",
+            type=str,
+            required=True,
+            help="The account name for the cloud cache.",
+        )
+        sub_parser.add_argument(
+            "--container",
+            type=str,
+            required=True,
+            help="The container name for the cloud cache.",
+        )
+        sub_parser.add_argument(
+            "--model_hash",
+            type=str,
```

Comment:
How would user know which model_hash to remove?

Reply:
User has to check the Storage blob or workflow log by himself to get the model hash. There is no way to directly get the model hash. Another option in my head is to have a config file as input, a specific command to run this workflow, but instead of "running the workflow in order to get the output model", this run will delete all model cache about this config in the cloud cache.